app/components/code-mirror.ts

```diff
@@ -259,6 +258,7 @@ const OPTION_HANDLERS: OptionHandlersSignature = {
   indentUnit: (indentUnitText) => (indentUnitText !== undefined ? [indentUnit.of(indentUnitText)] : []),
   indentWithTab: (enabled) => (enabled ? [keymap.of([indentWithTab])] : []),
   lineNumbers: (enabled) => (enabled ? [lineNumbers()] : []),
+  foldGutter: (enabled) => (enabled ? [foldGutter({ openText: '▼', closedText: '▶︎' }), keymap.of(foldKeymap)] : []),
```

Comment:
@VasylMarchuk is there a reason that this can't be sorted alphabetically like the others?

Reply:
Yep, turns out, the order in which extensions are fed to CodeMirror matters, and if `foldGutter` comes before `lineNumbers` — it appears so in the editor, making it look weird.